packages/sanity/src/core/create/studio-app/useStudioAppIdStore.ts

```diff
@@ -58,9 +58,9 @@ export function useStudioAppIdStoreInner(props: {
 
       try {
         const entry = await cache.get({projectId, appIdFetcher})
-        if (mounted) setAppId(entry?.appId)
+        if (mounted) setStudioApp(entry)
```

Comment:
This is mostly pure curiosity, but why make this change from sending in the id to sending in the whole object?

Reply:
The underlying api now returns studioApps as well, since we need it in dev mode. Before this was just an object with a single property, now it has two.